webpack/common.js

```diff
@@ -58,5 +58,12 @@ exports.convertKeysForHtml = function(obj){
 }
 
 exports.getVersion = function(){
-	return execSync("git describe --abbrev=7 --tags --dirty=+dirty --always", { encoding: 'utf8' }).toString();
+	try {
+		const { stdout } = execSync("git describe --abbrev=7 --tags --dirty=+dirty --always", { encoding: 'utf8' }).toString();
+		return stdout;
+	} catch (e) {
+		console.log("Something has happend when trying to get version");
```

Comment:
"Something has happend when trying to get version" ???
->
"Error when getting version from git"

Reply:
I would also add in message something that it doesn't affect build process in order not to disorient user about build itself.